small fixes to the short example func json template

PeerHerholz opened the pull request:
- deleted comma at the end of the last line
- changed RepetitionTime to must have a numerical value

Park-Patrick commented:
Hi @PeerHerholz thanks so much for the edits! Especially for editing both the short and full templates. 
The comma removal is correct, but could you please remove the square brackets for RepetitionTime? square brackets indicate an array in json files, while numbers should have no surrounding characters.

PeerHerholz commented:
Hi @Park-Patrick,

thanks for your message and hint. Super sorry for that, my bad!
I changed it accordingly. While doing so I checked the full example again and that also had square brackets for RepetitionTime, hence, I deleted those as well.
Thanks and sorry again. 
Best, Peer

Remi-Gau commented:
@all-contributors please add @Park-Patrick for review

Remi-Gau commented:
@all-contributors please add @PeerHerholz for code
